app/controllers/your-psp/post-toggle-worldpay-3ds-flex.controller.js

```diff
@@ -0,0 +1,26 @@
+'use strict'
+
+const paths = require('../../paths')
+const { renderErrorView } = require('../../utils/response')
+const { ConnectorClient } = require('../../services/clients/connector.client')
+
+module.exports = async (req, res) => {
```

Comment:
This function should have a name for clarity, e.g. ```module.exports = async toggleWorldpay3ds(req, res) { ... }```

Reply:
Thanks, I’ll do that.